python-threatexchange/threatexchange/exchanges/impl/tests/test_techagainstterrorism.py

```diff
@@ -0,0 +1,86 @@
+import pytest
+
+from threatexchange.exchanges.clients.techagainstterrorism import api
+from threatexchange.exchanges.impl.techagainstterrorism_api import (
+    TATSignalExchangeAPI,
+    TATCredentials,
+)
+from threatexchange.exchanges import fetch_state as state
+from threatexchange.exchanges.collab_config import (
+    CollaborationConfigWithDefaults,
+)
+
+
+def test_init():
+    api_instance = TATSignalExchangeAPI(username="test_user", password="test_pass")
+    assert api_instance.username == "test_user"
+    assert api_instance.password == "test_pass"
+
+
+def test_get_config_cls():
+    assert TATSignalExchangeAPI.get_config_cls() == CollaborationConfigWithDefaults
+
+
+def test_get_checkpoint_cls():
+    assert TATSignalExchangeAPI.get_checkpoint_cls() == state.NoCheckpointing
+
+
+def test_get_record_cls():
+    assert TATSignalExchangeAPI.get_record_cls() == state.FetchedSignalMetadata
+
+
+def test_get_credential_cls():
+    assert TATSignalExchangeAPI.get_credential_cls() == TATCredentials
+
+
+def test_get_name(monkeypatch):
+    monkeypatch.setattr(
+        "threatexchange.exchanges.impl.techagainstterrorism_api._API_NAME",
+        "test_api_name",
+    )
+    assert TATSignalExchangeAPI.get_name() == "test_api_name"
```

Comment:
blocking: This is a pretty tautological test. Why not just check `get_name()` against the string "tat" ?